Assets/Scripts/InventoryPlayer.cs

```diff
@@ -29,22 +31,29 @@ public class TestInventoryPlayer : MonoBehaviour
     [HideInInspector]
     public uint currentItemId = 0;
     private Vector2 _direction = new Vector2(0, 0);
-    private TestPlayer _testPlayer;
+    private Player _testPlayer;
     private Vector2 _lastDirectionToAttack = new Vector2(-1, -1);
+    int scaleSize = 5;
     // Start is called before the first frame update
     void Start()
     {
         bombAmount.text = "x" + _bombAmount;
+        if(rupeeAmount != null)
+        {
+            rupeeAmount.text = "x" + _rupeeAmount;
+        }
+        
         _playerAnimator = GetComponent<Animator>();
         _inventoryAnimator = panel.GetComponent<Animator>();
         _showInventory = false;
         _inventoryAnimator.updateMode = AnimatorUpdateMode.UnscaledTime;
         _inventoryAnimator.SetBool("isHidden", _showInventory);
         _showPanel = GetComponentInChildren<ShowPanel>();
         _inventory = GetComponentInChildren<Inventory>();
-        _testPlayer = GetComponentInChildren<TestPlayer>();
+        _testPlayer = GetComponentInChildren<Player>();
         _stackCounter = 0;
         _direction = Vector2.down;
+        transform.localScale = new Vector3(scaleSize, scaleSize, scaleSize);
```

Comment:
You don't need to scale on the Z axis.
This doesn't effect gameplay or usage of the inventory so you don't need to change it.